src/scripts/runscripts.sh

```diff
@@ -0,0 +1,89 @@
+#!/bin/bash
+HELP=false #Display help
+QUIET=false #Quiet mode
+COUNTER=0 # The count of tests run
+FAIL=0 # The count of failed tests
+PASS=0 # The count of passed tests
+while [[ $# -gt 0 ]]
+do
+key="$1"
+
+case $key in
+    --help)
+    HELP=true
+    shift
+    ;;
+    -q)
+    QUIET=true
+    shift
+    ;;
+    --test-folder)
+    TEST_FOLDER="$2"
+    shift
+    shift
+    ;;
+    --jerry-location)
+    JERRY_LOCATION="$2"
+    shift
+    ;;
+    *)
+    shift
+    ;;
+esac
+done
+if [ $HELP = true ] ; then
+	echo "Usage: runscripts.sh [OPTION]"
+	echo ""
+	echo "Possible options:"
+	echo "--help            display this help and exit"
+	echo "--q               only shows the final stats."
+	echo "--test-folder     the folder of the tests to run"
+	echo "                  required"
+	echo "--jerry-location  the location of jerryscript"
+	echo "                  required"
+	exit
+fi
+
+if [ -z $TEST_FOLDER ] ; then
+	tput setaf 1
+	echo "Test folder is required."
+	echo "Use --test-folder option!"
+	exit
+fi
+
+if [ -z $JERRY_LOCATION ] ; then
+	tput setaf 1
+	echo "Jerry location is required."
+	echo "Use --jerry-location option!"
+	exit
+fi
+
+for i in $(ls $TEST_FOLDER); do
```

Comment:
It might happen that the `ls` is too long, and `bash` cannot processes the results. You should check out `xargs` how to relax the number of arguments.